docs/openapi.yaml

```diff
@@ -101,6 +101,26 @@ paths:
         If using TypeScript, import typings for this response from our types package:
 
         `import type { TransactionResults } from '@blockstack/stacks-blockchain-api-types';`
+      parameters:
+        - name: limit
+          in: query
+          description: max number of transactions to fetch
+          required: false
+          schema:
+            type: integer
+        - name: offset
+          in: query
+          description: index of first transaction to fetch
+          required: false
+          schema:
+            type: integer
+        - name: type
+          in: query
+          description: Filter by transaction type
+          required: false
+          schema:
+            type: string
```

Comment:
This is type of `Array<string>`. Looks like the Express query array pattern is supported by openapi, 
e.g. `?type=coinbase&type=token_transfer`

https://swagger.io/docs/specification/serialization:
![image](https://user-images.githubusercontent.com/1447546/93223109-fdd13c00-f76f-11ea-8687-cf982f2ce1d8.png)

Reply:
So something like
```
type: array
items:
  type: string
  enum: [coinbase, token_transfer, smart_contract, contract_call, poison_microblock]
```

Reply:
Cool, updated!